### PR TITLE
Simplify the send screen, Moved button to the bottom of screen

### LINKED ISSUE #67: Simplify the receive screen
All reception screens should look the same. 
The ‘Advanced functions’ button should become unnecessary. 

Bildschirmfoto 2024-05-17 um 23 29 13

This change affects all wallets. Onchain, Lightning and MultiSig. 

The Advanced button should no longer be required. The selection of the quantity and a possible comment should be optionally defined on the first screen.

### LINKED ISSUE #68: Simplify the send screen.
All reception screens should look the same.
There should be no comment pop-up. 

Example:
I scan the QR code of a static Lightning address. I then get the following display:

Bildschirmfoto 2024-05-17 um 23 21 04

When I scan a static OnChain address or a Lightning address, I get the following:

Bildschirmfoto 2024-05-17 um 23 34 48

All screens should look the same. This means that when I scan a static Lightning invoice, the screen should look the same as when I scan an onChain address. 

The comment can be added directly or it can remain empty. No popup for the comment necessary.

### LINKED ISSUE #69: Show all buttons to continue at the bottom, not at the top.
right:

Bildschirmfoto 2024-05-17 um 23 42 31

wrong, the next button should be at the bottom, not at the top:
Bildschirmfoto 2024-05-17 um 23 43 27
Bildschirmfoto 2024-05-17 um 23 48 08

### LINKED ISSUE #79: Show a Max Button
Bildschirmfoto 2024-05-30 um 09 02 44

Implement a Max button
Do not show "MAX" but show the user the real number. In other words, the total balance. 

On the next screen you can see how many effective satellites arrive at the receiver address and how many satellites are used for the fee.

### PR DESCRIPTION
**Pull request by Danswar**
closes #67, closes #68 , closes #69, closes #79

### CODE REVIEW

**Comment by TaprootFreak** [comment]
Tested
Works as expected ✅

**Comment by Danswar** [comment]
Added #79 

IMG_E19DBBBCE0A0-1

File: screen/send/details.js [hunk 1]
```diff
@@ -609,16 +609,26 @@ const SendDetails = () => {
...
+    setAddresses(addrs => {
+      const currentBalance = wallet.getBalance();
+      addrs[scrollIndex.current].amountSats = currentBalance;
+      switch (units[scrollIndex.current]) {
+        case BitcoinUnit.SATS:
+          addrs[scrollIndex.current].amount = currency.satoshiToBTC(currentBalance);
+          break;
+        case BitcoinUnit.BTC:
+          addrs[scrollIndex.current].amount = currency.satoshiToBTC(currentBalance);
+          break;
```
Comment (davidleomay) [inline screen/send/details.js:625]: Why not summarize if it's the same logic?
```ts
        case BitcoinUnit.SATS:
        case BitcoinUnit.BTC:
          addrs[scrollIndex.current].amount = currency.satoshiToBTC(currentBalance);
          break;
```

**Comment by Danswar** [comment]
Added #67

Screenshot 2024-06-04 at 12 19 13 PM

File: screen/send/details.js (hunk 1 above)
Comment (Danswar) [reply screen/send/details.js]: done, thanks!

**Review by davidleomay — APPROVED** [review]

**Pull request merged**